src/Columns/IColumn.h

```diff
@@ -367,6 +376,25 @@ class IColumn : public COW<IColumn>
         throw Exception("Method structureEquals is not supported for " + getName(), ErrorCodes::NOT_IMPLEMENTED);
     }
 
+    static constexpr auto DEFAULT_ROWS_SEARCH_STEP = 8;
```

Comment:
These should probably go to ColumnSparse.h -- too specific for the general interface.